pkg/util/secretutil/secret_test.go

```diff
@@ -168,6 +169,53 @@ func TestGetCert(t *testing.T) {
 	}
 }
 
+func TestPfxGetCert(t *testing.T) {
```

Comment:
Instead of a new test, I would recommend adding a new table test in [`TestGetCert`](https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/main/pkg/util/secretutil/secret_test.go#L133)

Reply:
Same as other test explanations.